src/components/_video/VideoPlayer/VideoOverlay.tsx

```diff
@@ -10,21 +10,23 @@ import { getRandomIntInclusive } from '@/utils/number'
 import { EndingOverlay, ErrorOverlay, InactiveOverlay } from './VideoOverlays'
 import { PlayerState } from './VideoPlayer'
 
-type VideoOverlaProps = {
+type VideoOverlayProps = {
   playerState: PlayerState
   onPlay: () => void
   channelId?: string
   currentThumbnailUrl?: string | null
   videoId?: string
   isFullScreen?: boolean
+  playRandomVideoOnEnded?: boolean
```

Comment:
You haven't provided a default for that so now regular playback will also not have this functionality, right? Let's change props so that this is enabled by default. Actually, I think we could just introduce an `embedded` prop  to the video player instead, we may want to change some other things based on that as well in the future.